assets/css/style.css

```diff
@@ -270,13 +270,19 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    background-color: rgba(65,65,65,.75);
-    min-height: 33px;
+
     pointer-events: none;
     color: white;
-    padding-inline: 8%;
 }
 
+.cmt-widget .inner-address-bar {
+    display: flex;
+    padding-inline: 8%;
+    min-height: 33px;
+    justify-content: center;
+    align-items: center;
+    background-color: rgba(65, 65, 65, .75);
+}
```

Comment:
Address bar collapses when there's no address. Avoids the little grey box floating around